src/components/General/RandomBanner.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <span/>
+</template>
+
+<script>
```

Comment:
Could you refactor a bit to share more code between RandomPicture and RandomBanner?